pkg/version/version.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package version
+
+// Version holds the complete version number. Filled in at linking time.
+var Version = "1.11.0+unknown"
```

Comment:
Do we need to keep updating this `Version`?

Reply:
That's how containerd does it. We can keep this as a simple `unknown` string. :)

Reply:
This looks like a placeholder so I just wondered what we're going to do with it

Reply:
I'll change this to `unknown`.